yarn.lock

```diff
@@ -2843,6 +2843,11 @@
     google-protobuf "^3.17.0"
     long "^4.0.0"
 
+"@shapeshiftoss/types@3.1.3":
```

Comment:
run yarn and update this

Reply:
This was updated by yarn automatically. My assumption is due to the locked types version it decides to track it I guess? Typically we have not seen `^versions` tracked when included as peer/dev dependency which threw us off as well. I think it will be cleaned up after all packages have been updated